### ISSUE TITLE
add `alt="Description of the image"` of ALL images

### ISSUE THREAD

**Issue by flexbox** [MatthysDucrocq/tpResponsiveHtml#1]
https://github.com/MatthysDucrocq/tpResponsiveHtml/blob/4b105454164b76a735f92d452a49c4637158914c/article.html#L28
@MatthysDucrocq 

- [ ] all `img` tags must have `alt=`

**why?**

1. Adding alternative text to photos is first and foremost a principle of web accessibility. Visually impaired users using screen readers will be read an alt attribute to better understand an on-page image.
2. Alt text will be displayed in place of an image if an image file cannot be loaded.
3. Alt text provide better image context/descriptions to search engine crawlers, helping them to index an image properly.

👉 https://moz.com/learn/seo/alt-text
